src/titlepane/TitlePane.ts

```diff
@@ -108,36 +132,19 @@ export default class TitlePane extends TitlePaneBase<TitlePaneProperties> {
 		}, [
 			v('div', {
 				'aria-level': headingLevel ? String(headingLevel) : null,
-				classes: this.classes(
-					closeable ? css.closeable : null,
-					css.title
-				).fixed(
-					closeable ? css.closeableFixed : null,
-					css.titleFixed
-				),
-				onclick: this._onTitleClick,
-				onkeyup: this._onTitleKeyUp,
+				classes: this.classes(css.title, ...this.getModifierClasses()).fixed(css.titleFixed, ...this.getFixedModifierClasses()),
 				role: 'heading'
 			}, [
-				v('div', {
+				v('button', {
 					'aria-controls': this._contentId,
-					'aria-disabled': closeable ? null : 'true',
 					'aria-expanded': String(open),
+					disabled: !closeable,
 					classes: this.classes(css.titleButton),
 					id: this._titleId,
-					role: 'button',
-					tabIndex: closeable ? 0 : -1
+					onclick: this._onTitleClick
 				}, [
-					v('i', {
```

Comment:
Why isn't this displayed anymore?

Reply:
it is, I just moved it to `this.renderExpandIcon()`